docs/website/docs/developers/general/contributing.md

```diff
@@ -65,6 +65,9 @@ Signed-off-by: Random J Developer <random@developer.example.org>
     git commit -s -m 'This is my commit message'
     ```
 
+  See `.git/hooks/prepare-commit-msg.sample` for an example of automaticall
+  adding as part of hook to prepare commit message.
```

Comment:
Fix typo + some rewording suggestions

```suggestion
  See `.git/hooks/prepare-commit-msg.sample` for an example of automatically
  adding this using a [git hook](https://git-scm.com/docs/githooks) to prepare
  commit messages.
```

Also, my local file has
```
# SOB=$(git var GIT_COMMITTER_IDENT | sed -n 's/^\(.*>\).*$/Signed-off-by: \1/p')
# git interpret-trailers --in-place --trailer "$SOB" "$COMMIT_MSG_FILE"
# if test -z "$COMMIT_SOURCE"
# then
#   /usr/bin/perl -i.bak -pe 'print "\n" if !$first_line++' "$COMMIT_MSG_FILE"
# fi
```

but I might also try https://stackoverflow.com/a/46536244

Reply:
Tested the perl version in the local file, works on my Windows machine from CLI and `git gui`.

I'll also try commit signing 
* https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits
* https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key
* https://docs.github.com/en/authentication/managing-commit-signature-verification/associating-an-email-with-your-gpg-key
* https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key

Reply:
Commit signing works for me (after some OS-specific setup). The GitHub docs are pretty comprehensive, but we can condense them for common scenarios.